src/Jira/Api.php

```diff
@@ -347,6 +348,33 @@ public function getVersions($projectKey)
     }
 
     /**
+     * Helper method to find a specific version based on the name of the version.
+     *
+     * @param string $projectKey Project Key
+     * @param string $name The version name to match on
+     *
+     * @return int|null VersionId on match or null when there is no match
+     */
+    public function findVersionByName($projectKey, $name)
```

Comment:
I think we should add these methods since they are pretty basic functionality for anyone who wants to automate the release process using using JIRA Versions/Releases.